app/src/main/java/com/p2p/meshify/ui/components/PhysicsSwipeToDelete.kt

```
@@ -25,204 +25,43 @@ import androidx.compose.ui.unit.Dp
-                                if (offsetX.value.absoluteValue < unlockThresholdPx && newOffset.absoluteValue >= unlockThresholdPx) {
-                                    haptics.perform(HapticPattern.Pop)
-                                }
-                            }
-                        }
-                    )
-                }
-        ) {
-            content()
-        }
+        Box(Modifier.fillMaxWidth().offset { IntOffset(offsetX.value.roundToInt(), 0) }.clip(shape).background(MaterialTheme.colorScheme.surface).pointerInput(Unit) {
+            detectHorizontalDragGestures(onDragStart = { isDragging = true }, onDragEnd = { isDragging = false; scope.launch { if (!deleteEnabled || offsetX.value.absoluteValue < unlockThresholdPx) { if (offsetX.value.absoluteValue > 10f) haptics.perform(HapticPattern.Thud); offsetX.animateTo(0f, spring(0.55f)); isUnlocked = false } else { if (!isUnlocked) haptics.perform(HapticPattern.Pop); isUnlocked = true; offsetX.animateTo(-revealDistancePx, spring(0.6f)) } } }, onHorizontalDrag = { change, dragAmount -> change.consume(); if (dragAmount < 0 || offsetX.value < 0) { val friction = if (offsetX.value.absoluteValue < unlockThresholdPx && !isUnlocked) dragFriction * (1f - magneticPullStrength * (offsetX.value.absoluteValue / unlockThresholdPx)) else dragFriction; val newOffset = (offsetX.value + dragAmount * friction).coerceIn(-revealDistancePx * 1.2f, 0f); scope.launch { offsetX.snapTo(newOffset) }; if (offsetX.value.absoluteValue < unlockThresholdPx && newOffset.absoluteValue >= unlockThresholdPx) haptics.perform(HapticPattern.Pop) } })
```

detekt (1.23.8)

[warning] This else block is empty and can be removed.

(detekt.empty-blocks.EmptyElseBlock)